website/content/docs/api-gateway/usage/basic-usage.mdx

```diff
@@ -0,0 +1,135 @@
+---
+layout: docs
+page_title: Consul API Gateway Basic Usage
+description:  >-
+  Consul API Gateway Basic Usage
+---
+
+
```

Comment:
Since this is the only item in "Usage", can we flatten it and just call this one "usage"?

<img width="272" alt="image" src="https://user-images.githubusercontent.com/7556325/182292114-e18346c0-ce0d-4660-aac3-6f67ec5bd12d.png">

Reply:
d23cff615f5bd3a0ae99ef3895dcbb41cba0c5e8